apps/dashboard-service/src/dashboard/dto/cache.service.ts

```diff
@@ -0,0 +1,83 @@
+import { Injectable } from "@nestjs/common";
+import { Queue } from "bullmq";
+import { InjectQueue } from "@nestjs/bullmq";
+import { DashboardQueryDto } from "./dashboard-query.dto";
+
+import Redis from "ioredis";
+import Cache from "ioredis-cache";
```

Comment:
Ooh neat, I didn't find this library when I was doing the data api integration, and just rolled my own cache. I'll have to look into using it there 👍🏻

Reply:
Although now that I read closer; are you actually using this cache library? It looks like all your uses are just pulling the redis instance and using it directly.